tests/unit/experimental/pipeline/test_orchestrator.py

```diff
@@ -181,8 +189,8 @@ async def test_orchestrator_aggregation(pipeline_aggregation: Pipeline) -> None:
 async def test_orchestrator_aggregation_waiting(pipeline_aggregation: Pipeline) -> None:
     orchestrator = Orchestrator(pipeline=pipeline_aggregation)
     node_a = pipeline_aggregation.get_node_by_name("a")
-    node_a.status = RunStatus.DONE
+    node_a.status = {orchestrator.run_id: RunStatus.DONE}
     node_b = pipeline_aggregation.get_node_by_name("a")
-    node_b.status = RunStatus.UNKNOWN
+    node_b.status = {orchestrator.run_id: RunStatus.UNKNOWN}
```

Comment:
When does a `run_id` get set to `RunStatus.UNKNOWN`?

Reply:
Not needed indeed, removing.